views/partials/prs.hbs

```diff
@@ -75,6 +75,6 @@
 
     <div class="br2 center mt5 overflow-hidden w-50-ns w-80-m w-90 mt2">
         <button class="bg-orange white center mt2 h2 bn pointer br1 pv0 ph3 db-ns" id="saveUser">This is Me</button>
-        <p class='light-silver center tc-ns'>In the future, you can find your PRs by visiting <a href='{{ hostname }}/me' class='link white underline-hover'>{{ hostname }}/me</a> on this device.</p>
+        <p class='light-silver center tc-ns'>In the future, you can find your PRs by visiting <a href='{{ hostname }}/me' class='link white underline-hover' id="saveUser">{{ hostname }}/me</a> on this device.</p>
```

Comment:
As it's bad practice to re-use ids I think we should change this, perhaps we can put the id on the surrounding div and then target the button and the a tag from that or something? What do you think?

Reply:
I didn't think of that. Good point. Adding the ID to the surrounding div makes the entire "In the future,..." text clickable, which absolutely works but is a weird UX. Would it be OK to change saveUser into a class and make the click handler work on that class? I've tested that this works, but I'm unsure if this is also bad practice.

Reply:
Was thinking you could just target the button and the a, e.g. `$('#saveUser').find('button, a')` or something (note: pseudo code). But yeah, use class if easier 👍

Reply:
The latest commits use the class method I described.